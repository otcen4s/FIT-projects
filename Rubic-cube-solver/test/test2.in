112
112
112
226 333 144 555
226 333 144 555
226 333 144 555
664
664
664