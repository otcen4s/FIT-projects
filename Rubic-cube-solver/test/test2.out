112
112
112
226 333 144 555
226 333 144 555
226 333 144 555
664
664
664

111
111
111
222 333 444 555
222 333 444 555
222 333 444 555
666
666
666

